eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs

```diff
@@ -138,7 +139,14 @@ instance Crypto c => TranslateEra (ConwayEra c) EpochState where
         }
 
 instance Crypto c => TranslateEra (ConwayEra c) DState where
-  translateEra _ DState {..} = pure DState {..}
+  translateEra _ DState {dsUnified = umap, ..} = pure DState {dsUnified = umap', ..}
+    where
+      umap' =
+        umap
+          { UM.umElems =
+              Map.map (\(UM.UMElem rd _ poolId drep) -> UM.UMElem rd mempty poolId drep) (UM.umElems umap)
+          , UM.umPtrs = mempty
+          }
```

Comment:
We could create the UMap directly, 
```
      umap' = UM.UMap {
        UM.umElems = Map.map (\(UM.UMElem rd _ poolId drep) -> UM.UMElem rd mempty poolId drep) (UM.umElems umap),
        UM.umPtrs = mempty
      }
```
just because we're modifying both fields, but of course it makes no difference in the end.

Reply:
Yeah, I do agree with your suggestion, but I am gonna keep it this way, since I am too lazy to change it and this code will have to change later anyways, once we implement a Conway specific version of the UMap